eng/Versions.props

```diff
@@ -66,5 +62,13 @@
     <MicrosoftEntityFrameworkCoreSqlServerPackageVersion>8.0.8</MicrosoftEntityFrameworkCoreSqlServerPackageVersion>
     <MicrosoftEntityFrameworkCoreToolsPackageVersion>8.0.8</MicrosoftEntityFrameworkCoreToolsPackageVersion>
     <MicrosoftNETRuntimeWorkloadTestingInternalVersion>9.0.0-preview.5.24272.3</MicrosoftNETRuntimeWorkloadTestingInternalVersion>
+    <!-- needed to get 8.0 version for running tests -->
+    <MicrosoftNETCoreAppRefVersion>8.0.8</MicrosoftNETCoreAppRefVersion>
```

Comment:
Assuming this flows when using `darc update-dependencies`? Mainly asking as most of the versions above have the suffix `PackageVersion` so I wasn't aware that `Version` was enough.

Reply:
My mistake, Good catch, thanks!